packages/wisp-website/src/app/sentry-example-page/page.tsx

```
@@ -0,0 +1,225 @@
+      <Head>
+        <title>sentry-example-page</title>
+        <meta content="Test Sentry for your Next.js app!" name="description" />
+      </Head>
+      <main>
+        <div className="flex-spacer" />
+        <svg
+          fill="none"
+          height="40"
+          width="40"
+          xmlns="http://www.w3.org/2000/svg"
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)